apps/fulfillment-e2e/src/integration/partialy-picking-picklist.cy.ts

```diff
@@ -0,0 +1,158 @@
+import { PickingListsFragment } from '../support/page_fragments/picking-lists.fragment';
+import { PickingProductFragment } from '../support/page_fragments/picking-product.fragment';
+import { PickingFragment } from '../support/page_fragments/picking.fragment';
+import { PickingPage } from '../support/page_objects/picking.page';
+
+const pickingListId = '37cb241e-f18a-5768-985c-a2d7aff4875e';
+
+const pickingPage = new PickingPage(pickingListId);
+
+const pickingListsFragment = new PickingListsFragment();
+const pickingProductFragment = new PickingProductFragment();
+const pickingFragment = new PickingFragment();
+
+describe('Partial picking a picklist', () => {
+  beforeEach(() => {
+    cy.clearIndexedDB();
+    cy.login();
+    pickingPage.visit();
+  });
+
+  it('should check partial picking', () => {
+    // Setting initial number of products in each tab
+    pickingPage.insideTabContent(pickingPage.getNotPickedTab(), () => {
+      pickingProductFragment.getProducts().then((products) => {
+        cy.wrap(products.length).as('initialNotPickedListItemsNumber');
+      });
+    });
+    pickingPage.insideTabContent(pickingPage.getPickedTab(), () => {
+      pickingProductFragment.getProducts().should('not.exist');
+      cy.wrap(0).as('initialPickedListItemsNumber');
+    });
+    pickingPage.insideTabContent(pickingPage.getNotFoundTab(), () => {
+      pickingProductFragment.getProducts().should('not.exist');
+      cy.wrap(0).as('initialNotFoundListItemsNumber');
+    });
+
+    // For the first item, select only part of the quantity.
+    pickingPage.insideTabContent(pickingPage.getNotPickedTab(), () => {
+      pickingProductFragment
+        .getProducts()
+        .eq(0)
+        .within(() => {
+          pickingProductFragment.getQuantityInputPlusButton().click();
+          pickingProductFragment.getSubmitButton().click();
+        });
+    });
+
+    // Verify that the Partial Picking Modal is opened.
+    pickingFragment.getPartialPickingDialog().should('have.attr', 'open');
```

Comment:
`.should('have.attr', 'open');` - is not a good check-in E2E, because HTML can have this attribute, but it does not prove that the element is visible

let's change it to `.should('be.visible');` - that's what we want to see here, right?